modules/gke-cluster-standard/variables.tf

```diff
@@ -135,6 +135,7 @@ variable "enable_features" {
     fqdn_network_policy  = optional(bool, false)
     gateway_api          = optional(bool, false)
     groups_for_rbac      = optional(string)
+    image_streaming      = optional(bool, false)
```

Comment:
I'm a bit torn on whether we should stick to the provider naming (`gcfs`), but this is probably the right way to do it.